### DISCUSSION TITLE
ableton-live-suite11 deleted - what now?

### DISCUSSION THREAD

**Discussion by FlavioB79** [Homebrew/discussions#5339] [Casks]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.2.20
ORIGIN: https://github.com/Homebrew/brew
HEAD: c2ed3327c605c3e738359c9807b8f4cd6fec09eb
Last commit: 7 days ago
Core tap JSON: 05 May 16:26 UTC
Core cask tap JSON: 05 May 16:26 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 3.1.4 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.1.4/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 15.0.0 build 1500
Git: 2.39.3 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 8.4.0 => /usr/bin/curl
macOS: 14.4.1-arm64
CLT: 15.3.0.0.1.1708646388
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

Hi.
Today running "brew upgrade --greedy" told me that ableton-live-suite11 cask would be deleted. OK, so now I see that there's ableton-live-suite@11 in the cask list: how do I have to proceed, to have my Ableton Live 11 always updated? Can I simply run "brew install ableton-live-suite@11" or will this clash with the existing "ableton-live-suite11"?
Thanks!

### Relevant casks

ableton-live-suite11, ableton-live-suite@11

**Comment by SMillerDev** [comment]
If it's only deleted something might be going wrong. Try doing a manual `brew update` since the moderation should be automatic

**Reply by FlavioB79** [reply]
So you mean just "brew update ableton-live-suite11"? Or "brew update ableton-live-suite@11"?

Edit: "brew update" just works for homebrew itself, it doesn't accept names afterward. What exactly would you want me to do? I issued "brew update" and all I got was this:

==> Updating Homebrew...
Already up-to-date.

**Reply by SMillerDev** [reply]
I meant just `brew update`.

**Reply by FlavioB79** [reply]
That's what I did and I got "Already up-to-date".
Now, if I check the list of installed software, I still see the "old" writing for Ableton Live:

brew list's output (only the relevant part):

==> Casks
ableton-live-suite11	logi-options-plus	sysex-librarian

**Reply by FlavioB79** [reply]
Oh, ok it seems all good:

user@Mac-mini ~ % brew install ableton-live-suite11
Warning: Formula homebrew/cask-versions/ableton-live-suite11 was renamed to homebrew/cask/ableton-live-suite@11.
Warning: Not upgrading ableton-live-suite11, the latest version is already installed
